types/lodash/fp.d.ts

```diff
@@ -1789,7 +1789,7 @@ declare namespace _ {
     }
     type LodashIntersectionWith1x5<T1> = (array: lodash.List<T1> | null | undefined) => T1[];
     type LodashIntersectionWith1x6<T1, T2> = (comparator: lodash.Comparator2<T1, T2>) => T1[];
-    type LodashInvert = (object: object) => lodash.Dictionary<string>;
+    type LodashInvert = <T extends Record<PropertyKey, PropertyKey>>(object: T) => lodash.InvertResult<T>;
```

Comment:
FYI there is a bug in `generate-fp.ts`; this comes out as:

```ts
type LodashInvert = <T extends Record<PropertyKey>(object: T) => lodash.InvertResult<T>;
```

which is a syntax error. I had to manually edit.